devnet-sdk/devstack/presets/orchestrator.go

```diff
@@ -43,38 +39,44 @@ func DoMain(m *testing.M) {
 		Pid:    false,
 	})
 
-	t := stack.NewToolingT("Main", logger)
+	p := devtest.NewP(logger)
+	defer p.Close()
+
+	p.Require().NotEmpty(opts, "Expecting orchestrator options")
 
 	// For the global geth logs,
 	// capture them in the global test logger.
 	// No other tool / test should change the global logger.
 	// TODO(#15139): set log-level filter, reduce noise
 	//log.SetDefault(t.Log.New("logger", "global"))
 
-	initOrchestrator(t, t.Log)
+	initOrchestrator(p, opts...)
+
 	code := m.Run()
-	t.RunCleanup()
 	os.Exit(code)
```

Comment:
Sadly, this will exit without running the `defer p.Close()`.  `defer` often doesn't work if tests panic either because other places seem to be calling `os.Exit` on us.  Go makes it really hard to force something to run on shutdown....